app/views/incomeSources/manage/ManageObligations.scala.html

```diff
@@ -0,0 +1,210 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import _root_.implicits.ImplicitDateFormatterImpl
+@import views.html.layouts.unifiedLayout
+@import views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import models.incomeSourceDetails.TaxYear
+@import models.incomeSourceDetails.viewmodels.ObligationsViewModel
+
+@this(
+        mainTemplate: unifiedLayout,
+        implicitDateFormatter: ImplicitDateFormatterImpl,
+        continueButton: ContinueButton,
+        formWithCSRF: FormWithCSRF,
+        h1: h1,
+        h2: h2,
+        p: p,
+        link: link,
+        hr: hr,
+        govukTable: GovukTable
+)
+
+
+@(
+        sources: ObligationsViewModel,
+        businessName: String,
+        taxYear: TaxYear,
+        changeTo: String,
+        isAgent: Boolean,
+        backUrl: String,
+        postAction: Call
+)(implicit messages: Messages, user: auth.MtdItUser[_])
+
+    @import implicitDateFormatter.longDate
+
+    @getMessage(key: String, args: String*) = @{
+        messages(s"business-added.$key", args: _*)
+    }
+
+    @getUniqueMessage(key: String, args: String*) = @{
+        messages(s"incomeSources.add.manageObligations.$key", args: _*)
+    }
+
+    @getBusinessName(name: String) = @{
```

Comment:
I don't think you really need lines 59 and 60, you can remove these;
String is usually not the best choice to model enumeration, but I guess it really your choice here